Library/Koala/Koala.swift

```diff
@@ -2043,10 +2048,16 @@ public final class Koala {
   private func defaultProperties() -> [String: Any] {
     var props: [String: Any] = [:]
 
+    let enabledFeatureFlags = self.config?.features
+      .filter { _, v in v == true }
+      .keys
+      .sorted()
+
     props["manufacturer"] = "Apple"
     props["app_version"] = self.bundle.infoDictionary?["CFBundleVersion"]
     props["app_release"] = self.bundle.infoDictionary?["CFBundleShortVersionString"]
-    props["current_variants"] = AppEnvironment.current.config?.abExperimentsArray
+    props["current_variants"] = self.config?.abExperimentsArray.sorted()
```

Comment:
Whenever values from the config are accessed in other parts of this class, the object's reference of `config` was used. Grabbing the `config` from the `AppEnvironment` was a deviation from this pattern, so I updated it to be in-line with how the `config` is accessed elsewhere in this class. Please note that anytime the config is updated, it is also updated in the `Koala` instance stored in `AppEnvironment` so we don't have to worry about out-of-date config values: https://github.com/kickstarter/ios-oss/blob/master/Library/AppEnvironment.swift#L68